### PR TITLE
Search Listing

### PR DESCRIPTION
**Pull request by asdfkg**
Implement search listing page

### CODE REVIEW

File: reservations/index.php [hunk 1]
```diff
@@ -45,16 +43,22 @@
...
+    <script src="/js/react/search-result.jsx"  type="text/jsx"></script>
+    <!-- /React -->
 </head>
 
 <body>
 	<?php require_once '../inc-header.php'; ?>
         <!-- Contact Us Header Image Section Start -->
         <section id="header-section">
-            <img src="/villazzo_new/img/destination-header_<?php echo str_replace('-', '_', $_GET['dest']); ?>.png">
+            <img src="/img/destination-header_<?php echo str_replace('-', '_', $_GET['dest']); ?>.png">
```
Comment (nsrivastava010) [inline reservations/index.php]: Please use common tag for image

File: js/react/search-result.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            var dd = dt.getDate();
+            var mm = dt.getMonth()+1;
+            var y = dt.getFullYear();
+            return mm+'/'+ dd + '/'+ y;
+        };
+        var searchItemLeft = function(object){
+            return (<div className="medium-6 columns property-image" data-equalizer-watch>
+                        <span className="img-property-title">{object.property_name}</span>
+                        <a href={"/"+object.dest_name+"-rental-villas/villa-"+object.property_name.toLowerCase().replace(' ','-')+(object.check_in_dt && object.check_out_dt?'?check_in='+getFormattedDate(object.check_in_dt)+'&check_out='+getFormattedDate(object.check_out_dt):'')}>
+                            <Image src={object.property_img} />
```
Comment (nsrivastava010) [inline js/react/search-result.jsx]: Use Image tag

File: js/react/search-result.jsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                            }
+
+                            if (object.booked_till_dt){
+                                var propertyBook = (
+                                    <a className="button tiny expand" href={"?dest="+object.dest_name+"&check_in="+getFormattedDate(object.booked_till_dt,1)+"&check_out="+getFormattedDate(object.booked_till_dt,4)}>
+                                        AVAIL. {getFormattedDate(object.booked_till_dt,1)}
+                                    </a>
+                                )
+                            } else if(object.check_in_dt && object.check_out_dt){
+                                var propertyBook = <a className="button tiny expand" href={"/reservations/"+bookUrl+'?property='+object.property_name+'&check_in='+getFormattedDate(object.check_in_dt)+'&check_out='+getFormattedDate(object.check_out_dt)}>BOOK NOW</a>
```
Comment (nsrivastava010) [inline js/react/search-result.jsx]: please use common  anchor tag

File: js/react/search-result.jsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                            }else{
+                                var propertyBook = (
+                                        <a className="button tiny expand" href={'/reservations/?dest='+object.dest_name+'&check_in='+getFormattedDate(new Date())+'&check_out='+getFormattedDate(new Date(),3)}>BOOK NOW</a>
```
Comment (nsrivastava010) [inline js/react/search-result.jsx]: please use common  anchor tag

File: js/react/search-result.jsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+                                )
+                            }
+
+                            return <div className="small-12 columns property-spacer">
+                                    <div className="row collapse" data-equalizer>
+                                        {searchItemLeft(object)}
+                                        <div className="medium-6 columns property-details" data-equalizer-watch>
+                                            <div className="row text-center">
+                                                <div className="small-12 columns">
+                                                    <h2 className="text-grey" dangerouslySetInnerHTML={unescapeHTML(object.property_desc_short)}></h2>
```
Comment (nsrivastava010) [inline js/react/search-result.jsx]: Please define heading tag in common jsx file

File: js/react/search-result.jsx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+                                                </div>
+                                            </div>
+                                            <div className="row text-center">
+                                                    <div className="small-12 columns">
+                                                            <h2>{propertyRate}</h2>
+                                                    </div>
+                                            </div>
+                                            <div className="row text-center">
+                                                    <div className="small-4 small-offset-2 columns">
+                                                            <a href={"/"+object.dest_name+"-rental-villas/villa-"+object.property_name.toLowerCase().replace(' ','-')+(object.check_in_dt && object.check_out_dt?'?check_in='+getFormattedDate(object.check_in_dt)+'&check_out='+getFormattedDate(object.check_out_dt):'')} className="button tiny expand">DETAILS</a>
```
Comment (nsrivastava010) [inline js/react/search-result.jsx]: please use common  anchor tag

File: js/react/search-result.jsx (hunk 2 above)
Comment (asdfkg) [inline js/react/search-result.jsx]: Here we have image inside Anchor tag thatswhy this is not possible

**Pull request merged**